src/runtime/ui.config/forms/input.ts

```diff
@@ -77,7 +77,7 @@ export default {
     },
     leading: {
       wrapper: 'absolute inset-y-0 start-0 flex items-center',
-      pointer: 'pointer-events-none',
+      pointer: 'pointer-events-auto',
```

Comment:
This can't be removed, it would change the behavior of lots of apps: https://ui.nuxt.com/components/input#trailing